phpunit/functional/Glpi/Form/Tag/SectionTagProviderTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace tests\units\Glpi\Form\Tag;
+
+use DbTestCase;
+use Glpi\Form\AnswersSet;
+use Glpi\Form\Form;
+use Glpi\Form\Tag\Tag;
+use Glpi\Tests\FormBuilder;
+use Glpi\Tests\FormTesterTrait;
+
+final class SectionTagProviderTest extends DbTestCase
+{
+    use FormTesterTrait;
+
+    public function testGetTagsForFormWithoutSections(): void
+    {
+        $form = $this->createForm(new FormBuilder());
+        $this->checkTestGetTags($form, []);
+    }
+
+    public function testGetTagsForFormWithSections(): void
+    {
+        $form = $this->getFormWithSections();
+        $this->checkTestGetTags($form, [
+            new Tag(
+                label: 'Section: Personal information',
+                value: $this->getSectionId($form, 'Personal information'),
+                provider: \Glpi\Form\Tag\SectionTagProvider::class,
```

Comment:
Small note, a `use Glpi\Form\Tag\SectionTagProvider` statement would be more clean as you use this class of lot in this file.

Also note that the class have the "Test" suffix with PHPunit so we can import the related class without alias (unlike with atoum).